code/content_packages/ascent/datum/codex.dm

```diff
@@ -0,0 +1,34 @@
+/datum/codex_entry/ascent
+	display_name = "The Ascent"
+	associated_strings = list("ascent")
+	associated_paths = list(
+		/mob/living/silicon/robot/flying/ascent,
+		/obj/item/multitool/mantid,
+		/obj/item/weldingtool/electric/mantid,
+		/obj/item/cell/mantid,
+		/obj/item/gun/energy/particle,
```

Comment:
can you move these guns here too while you're at it?

Reply:
I thought I already did.

Reply:
https://github.com/NebulaSS13/Nebula/pull/161/files#diff-83e7ab0cf4664cd8fe84fa6099e23c58R1